code/modules/antagonists/changeling/powers/regenerate.dm

```diff
@@ -28,7 +28,9 @@
 			C.regenerate_limbs(1)
 		C.regenerate_organs()
 		if(!user.getorganslot(ORGAN_SLOT_BRAIN))
-			var/obj/item/organ/brain/changeling_brain/B = new()
+			var/obj/item/organ/brain/B = new()
```

Comment:
Can you grab a mutant brain type from the species datum instead? Or does that introduce bugs?